rotkehlchen/data/assets_collections.sql

```diff
@@ -0,0 +1,726 @@
+INSERT INTO asset_collections(id, name, symbol) VALUES 
```

Comment:
shouldn't we do both here and below `OR IGNORE` just to be safe?

Reply:
okey